salt/modules/cmdmod.py

```diff
@@ -1422,9 +1451,7 @@ def run_stderr(cmd,
                ignore_retcode=ignore_retcode,
                use_vt=use_vt,
                saltenv=saltenv,
-               pillarenv=kwargs.get('pillarenv'),
-               pillar_override=kwargs.get('pillar'),
```

Comment:
This will break. We're expecting the kwarg `pillar_override` in the implementation but what's coming in to the function is the `pillar` kwarg.